install-and-configure/advanced-configuration/resource-consumption.md

```diff
@@ -1,54 +1,113 @@
 # Tuning Resource Consumption
 
-Kubecost can run on clusters with thousands of nodes when resource consumption is properly tuned. Here's a chart with some of the steps you can take to tune Kubecost, along with descriptions of each.
+## Cost Model
 
-![Memory Reduction Steps](/images/resource-consumption.png)
+**Reduced ETL builds.** Reducing the ETL build window can reduce the amount of data Kubecost is requested to process, which can reduce resource consumption.
 
-## Disable Cloud Costs on secondary clusters
+{% hint style="warning" %}
+`etlHourlyStoreDurationHours` can be set to `0` if hourly cost granularity is not required in your environment. Note, that some features on the Savings page default to using hourly data, but can be toggled to use daily data.
+{% endhint %}
 
-[Cloud Costs](/install-and-configure/install/cloud-integration/README.md#cloud-costs) allow Kubecost to pull in spend data from your integrated cloud service providers.
-
-Cloud cost metrics for all accounts can be pulled in on your primary cluster by pointing Kubecost to one or more management accounts. Therefore, you can disable CloudCost on secondary clusters by setting the following Helm value:
+```yaml
+kubecostModel:
+  etlDailyStoreDurationDays: 7
+  etlHourlyStoreDurationHours: 0
+```
 
-`--set cloudCost.enabled=false`
+**Disable features which are default-enabled.** Features such as [Container Stats](/architecture/containerstats-pipeline.md) and [Network Monitoring](/using-kubecost/navigating-the-kubecost-ui/network-monitoring.md) are enabled by default. If not using these features, disabling them can reduce Prometheus queries, and also reduce memory consumption of cost-model and Prometheus.
+
+```yaml
+kubecostModel:
+  containerStatsEnabled: false
+networkCosts:
+  config:
+    services:
+      # These are typically enabled by default if the networkcost daemonset is also enabled
+      google-cloud-services: false
+      amazon-web-services: false
+      azure-cloud-services: false
+```
```

Comment:
This is not always necessary, but I've found to be helpful in some large environments.